PSKoans/Public/Show-Karma.ps1

```diff
@@ -80,6 +85,29 @@ function Show-Karma {
                 $KoanLocation | Invoke-Item
             }
         }
+        'OpenFile' {
+            try {
+                $Results = Get-Karma @GetParams
+            }
+            catch {
+                $PSCmdlet.ThrowTerminatingError($_)
+            }
+            $FilePath = Get-PSKoan -Topic $Results.CurrentTopic.Name -Scope User | Select-Object -ExpandProperty Path
+            $LineNumber = $Results.CurrentTopic.FailedLineNumber
+            if ($PSVersionTable.Platform -eq "win32nt") {
+                $Editor = (Get-ItemProperty -Path 'HKCU:\Software\Microsoft\Windows\CurrentVersion\Explorer\FileExts\.ps1\UserChoice').ProgId
+                $Editor =  [regex]::matches($DefaultProgram,'(?<=\\).+(?=.exe)').value
+                switch ($Editor) {
+                    {($_ -eq "code") -or ($_ -eq "code-insiders")} {$FullArgument = '-g "{0}":{1}' -f $FilePath, $LineNumber; Start-Process $Editor -ArgumentList $FullArgument}
```

Comment:
```suggestion
                    { $_ -in 'code', 'code-insiders' } {
                        $FullArgument = '--goto "{0}":{1} --reuse-window' -f $FilePath, $LineNumber
                        Start-Process $Editor -ArgumentList $FullArgument
                    }
```
Bit too much going on here for one line, I think! 😄 Probably doesn't hurt to use full parameter names for VS code so it's a bit easier on the eyes as well...

Reply:
Yep this is much better 😄